### PR TITLE
Medical Engine - Hide actions that have ACE interactions

### PR DESCRIPTION
**Pull request by Cyruz143**
**Remove the vanilla actions to unload unconcious**
- People seem to accidentally hit this when trying to mount up (I know they should be using quickmount...)
- ACE adds it's own interaction to unload people

Needs MP test (hence draft)

### CODE REVIEW

File: addons/medical_engine/CfgActions.hpp [hunk 1]
```diff
@@ -13,4 +13,32 @@ class CfgActions {
     class FirstAid: None {
         show = 0;
     };
+    class UnloadUnconsciousUnits: None
+    {
```
Comment (jonpas) [inline addons/medical_engine/CfgActions.hpp:17]: !!!

Comment (Cyruz143) [reply addons/medical_engine/CfgActions.hpp:17]: Haha, that's what I get for taking it out of vanilla configs

**Comment by Cyruz143** [comment]
Tested, no vanilla actions to unload unconcious from driver, cargo, turret or gunner.

**Review by veteran29 — APPROVED** [review]
LGTM, verfied no ubc errors? I recall that some vanilla actions do not inherit from `None`

**Comment by Cyruz143** [comment]
> 
> 
> LGTM, verfied no ubc errors? I recall that some vanilla actions do not inherit from `None`

Taken from AIO dump so inheritence should be all fine. No UBCs I can see.

**Comment by PabstMirror** [comment]
I noticed these actions when testing #7959 and I think we'll need them gone for that PR

**Review by BaerMitUmlaut — APPROVED** [review]

**Review by commy2 — APPROVED** [review]

**Comment by jonpas** [comment]
Many approvals, merging.

**Pull request merged**